### ISSUE TITLE
Crashing and Animations at 90 degrees off

### ISSUE THREAD

**Issue by tomhalpin8** [E1P3/Learned_Motion_Matching_UE5#1]
First off, Amazing work! I didn't expect to see anyone do learned motion matching in Unreal anytime soon! I really hope it becomes a standard. Now on to the crashes. It would crash as soon as I hit play. Then when looking at the animations, they all have the character face first in the floor, like he had been rotated. Tried fixing that but couldn't get it up right. Then also going to the folder with the ThirdPersonCharacter_BP, just opening that folder would crash Unreal. Thank you for putting this project out there! I hope to try it soon. You're demo video look promising!

**Comment by E1P3** [comment]
Thank you! I didnt realise anyone would raise an issue so sorry for the late reply. The animations being rotated like that is mainly because they were originally in y+ axis so i had to manually shift them 90 degrees. There is probably a better way to convert them but i was rushing to the deadline of my master thesis.  Most of the crashes should come from binaries of normalization data not being where they are supposed to. If you see this would you mind sharing the crash report you got?

**Comment by tomhalpin8** [comment]
It's been awhile since I opened your project had to unzip it again. I compiled the binaries, the project opened, but hitting play crashed it. Thank you for getting back! 
[Testing.log](https://github.com/user-attachments/files/16617578/Testing.log)

**Comment by tomhalpin8** [comment]
> It's been awhile since I opened your project had to unzip it again. I compiled the binaries, the project opened, but hitting play crashed it. Thank you for getting back! [Testing.log](https://github.com/user-attachments/files/16617578/Testing.log)


https://github.com/user-attachments/assets/5776c19d-61f9-47cd-ac40-f64d1fe977b7

[Testing.log](https://github.com/user-attachments/files/16617654/Testing.log)

**Comment by E1P3** [comment]
Ok i see now. I downloaded the code again and tried to run in in development mode and indeed the file parser that loads the feature mean and scale crashes for some reason.  I was doing most of my work in debug through vscode in debug mode  that's why i havent encountered the same problem. When I get some time I will just add a denormalization layer to the model so i dont need to parse the file at all. For now though you can just run the project from the IDE (Visual Studio or vsc). The project does not crash then. 

Basically Edit -> Generate Visual Studio Project and then run it from the IDE

Hope that helps
